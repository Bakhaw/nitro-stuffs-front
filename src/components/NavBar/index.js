import React from 'react';
import { Link } from 'react-router-dom';

const routes = [
  {
    text: 'Home',
    to: '/'
  },
  {
    text: 'Maps',
    to: '/maps'
  }
];

function NavBar() {
  return (
    <nav>
      <ul>
        {routes.map(({ text, to }, index) => (
          <Link key={index} to={to}>
            {text}
          </Link>
        ))}
      </ul>
    </nav>
  );
}

export default NavBar;
