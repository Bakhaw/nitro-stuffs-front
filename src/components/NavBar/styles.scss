.NavBar {
  position: fixed;
  display: flex;
  flex: 1;
  background: $Blue;
  width: $NavBar__Width;
  height: $NavBar__Height;
  .NavBar_Links {
    display: flex;
    flex: 1;
    flex-direction: column;
    align-items: center;
    padding-top: 35px;
    .NavBar_Link {
      padding-bottom: 45px;
      .NavBar_img {
        width: 60px;
        height: 60px;
      }
      .NavBar_img:hover {
        transform: scale(1.1);
      }
    }
  }
}
