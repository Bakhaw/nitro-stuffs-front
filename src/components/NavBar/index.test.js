import React from 'react';
import { shallow } from 'enzyme';

import NavBar from '.';

test('should render properly', () => {
  const container = shallow(<NavBar />);
  expect(container.debug()).toMatchSnapshot();
});
