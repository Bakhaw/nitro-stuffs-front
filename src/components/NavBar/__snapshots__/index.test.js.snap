// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
"<nav>
  <ul>
    <Link to=\\"/\\">
      Home
    </Link>
    <Link to=\\"/maps\\">
      Maps
    </Link>
  </ul>
</nav>"
`;
